src/applications/personalization/dashboard/components/PrescriptionCard.jsx

```diff
@@ -34,59 +26,22 @@ export default function PrescriptionCard({ prescription }) {
           ? 'We’ve shipped your order'
           : 'We’re working to fill your prescription'}
       </p>
-      <p>
-        <strong>You submitted your refill order on:</strong>{' '}
-        {formatDate(refillSubmitDate || refillDate, {
-          format: 'L',
-        })}
-      </p>
-      <p>
-        {
-          // Disabling interactivity.
-          // See: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14499
-          //   isTrackable ? (
-          //   <Link
-          //     key={`rx-${prescription.id}-track`}
-          //     className="rx-track-package-link usa-button"
-          //     href={`/health-care/prescriptions/${prescription.id}/track`}
-          //     onClick={recordDashboardClick('track-your-package')}
-          //   >
-          //     Track Your Package
-          //   </Link>
-          // ) : (
-          //   <Link
-          //     className="usa-button usa-button-primary"
-          //     href={`/health-care/prescriptions/${prescription.id}`}
-          //     onClick={recordDashboardClick('view-your-prescription')}
-          //   >
-          //     View Your Prescription
-          //     <i className="fa fa-chevron-right" />
-          //   </Link>
-          //   )
-        }
-      </p>
+      {dateInPast && (
+        <p>
+          <strong>You submitted your refill order on:</strong>{' '}
+          {submittedDate.format('L')}
+        </p>
+      )}
     </div>
   );
 }
 
 PrescriptionCard.propTypes = {
   prescription: PropTypes.shape({
-    id: PropTypes.string.isRequired,
```

Comment:
I removed the prop types that aren't used in this component, because it seems wrong for it to set requirements around data it doesn't care about.